frontend/src/components/marketing/marketing.css

```
@@ -0,0 +1,670 @@
+  color: var(--ppm-slate-300);
+  font-size: 0.95rem;
+  text-decoration: underline;
+  text-decoration-color: transparent;
+  text-underline-offset: 0.2em;
+  transition: color 160ms ease, text-decoration-color 160ms ease;
+}
+
+.ppm-hero-link:hover,
+.ppm-hero-link:focus-visible {
+  color: var(--color-primary-foreground);
+  text-decoration-color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)